cypress/e2e/pages/assets.cy.ts

```diff
@@ -0,0 +1,89 @@
+/// <reference types="cypress" />
+
+const assetsTypes = ['Fungible', 'NonFungible'];
+
+const assets_links: { name: string; link: string }[] = [];
+
+describe('Assets Page', () => {
+  beforeEach(() => {
+    cy.visit('/assets');
+  });
+
+  it('should load the assets page', () => {
+    cy.get('h1', { timeout: 10000 }).contains('Assets').should('be.visible');
+  });
+
+  assetsTypes.forEach(type => {
+    if (typeof type !== 'string') return;
+
+    it(`should filter Assets by type - ${type}`, () => {
+      cy.wait(5000);
+
+      const typeFilter = cy.get(':nth-child(2) > [data-testid="selector"]', {
```

Comment:
highly coupled test with dom selectors. Place the dom strings into isolated variables for better maintenance.